misc/determine-version.py

```diff
@@ -1,11 +1,13 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
```

Comment:
This change seems like it could break it for others.

Reply:
On the Linux boxen I've checked, the python package installed python2.7 as the actual executable and python2 and python as links to it.  I could be wrong.

Reply:
The RPMs that I've reviewed install python2.7 as the actual file with python2 and python as symlinks to it.

Reply:
I think leaving it as 

```
#!/usr/bin/env python
```

And making sure we are compatible with both 2.7 and 3+ is the safest way.

Reply:
> And making sure we are compatible with both 2.7 and 3+ is the safest way.

:+1: